app/javascript/plugins/call_tool/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { omit, pick } from 'lodash';
+import * as React from 'react';
+import { render } from 'react-dom';
+import { Store } from 'redux';
+import ComponentWrapper from '../../components/ComponentWrapper';
+import { IAppState } from '../../types';
+import { camelizeKeys } from '../../util/util';
+import Plugin from '../plugin';
+import CallToolView from './CallToolView';
+
+interface ICallToolOptions {
+  el: HTMLElement;
+  namespace: string;
+  config: any; // todo
+  store: Store<IAppState>;
+}
+export const init = options => {
+  if (!options.el) {
+    options.el = document.getElementById('call-tool-component');
+  }
+  const { el, config, store } = options;
+  const member = window.champaign.personalization.member;
+  const memberData = pick(member, 'name', 'email', 'country', 'postal');
+  const formValues = window.champaign.personalization.formValues;
+  return new CallTool({
+    config: {
+      ...config,
+      ...memberData,
+      ...formValues,
+    },
+    el,
+    namespace: 'calltool',
+    store,
+  });
```

Comment:
When we do:

```ts
new CallTool({
    config: {...}, // This needs to match `data` from the _call_tool partial
    el,
    namespace: 'calltool',
    store,
  });
```

We need to make the `config` object match `data` from here: https://github.com/SumOfUs/Champaign/pull/1439/commits/37b3c24253103f242c9a160cfe09a3c70062bc34#diff-44620064a03cc54d46d80f5d09569958L18-L31